pkgs/games/sfrotz/default.nix

```diff
@@ -0,0 +1,72 @@
+{
+  fetchFromGitLab,
+  freetype,
+  libao,
+  libjpeg,
+  libmodplug,
+  libpng,
+  libsamplerate,
+  libsndfile,
+  libvorbis,
+  pkg-config,
+  SDL2,
+  SDL2_mixer,
+  stdenv,
+  zlib
+}:
+
+stdenv.mkDerivation rec {
+  pname = "sfrotz";
+  version = "2.52";
+
+  src = fetchFromGitLab  {
+    domain = "gitlab.com";
+    owner = "DavidGriffith";
+    repo = "frotz";
+    rev = version;
+    sha256 = "11ca1dz31b7s5vxjqncwjwmbbcr2m5v2rxjn49g4gnvwd6mqw48y";
+  };
+
+  buildInputs = [
+    freetype
+    libao
+    libjpeg
+    libmodplug
+    libpng
+    libsamplerate
+    libsndfile
+    libvorbis
+    pkg-config
+    SDL2
+    SDL2_mixer
+    stdenv
```

Comment:
Seeing stdenv as a buildInput here is surprising, have you checked whether that's really needed?

Reply:
This slipped in by mistake while I was rearranging things for readability. Removed.